.github/workflows/build.yaml

```diff
@@ -104,7 +104,9 @@ jobs:
       - "markdown-lint"
     steps:
       - uses: actions/checkout@v3
-      - uses: mobilecoinfoundation/actions/dcap-libs@main
+      - uses: mobilecoinfoundation/actions/dcap-libs@dcap_libs_version
```

Comment:
similar to #213 this should go back to `@main` once https://github.com/mobilecoinfoundation/actions/pull/3 comes in.